sdk/containerregistry/container-registry/src/containerRegistryUserCredentialPolicy.ts

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This credential is added for easier SDK development. It will be removed from beta.1 so don't need to review it too carefully. #14172

Reply:
Also curious, is this something we plan to support by GA, or are we just pushing for TokenCredential? (I'm fine with the latter if that's the case!)

Reply:
We will only support AAD. The service is also deprecating basic auth for security reason.